### DISCUSSION TITLE
How to add custom UI elements to service

### DISCUSSION THREAD

**Discussion by benlevinNEU** [RavenSystem/esp-homekit-devices#1884] [Q&A] [answered]
Is it possible to add multiple sliders and a button pad to the UI of a service in HomeKit?

I sort of asked this in my last post but I asked a bit too much and it was locked. I just want to know if this is possible.

**Comment by RavenSystem** [comment, marked as answer]
And I said, this site is for HAA firmware questions, not for HomeKit development questions.
